src/App.js

```diff
@@ -26,7 +26,7 @@ function App() {
         />
```

Comment:
All references to [category, setCategory] can be removed because you're now using useParams. So you can delete line 14 in this file and everything related to it in Categories etc., including the onClick in your categories file.